apps/docs-app/docs/features/api/og-image-generation.md

```diff
@@ -41,8 +41,6 @@ Next, define an API route in the `src/server/routes` directory.
 
 ```ts
 // src/server/routes/v1/og-images.ts
-import { defineEventHandler, getQuery } from 'h3';
```

Comment:
I'm on the fence with this one. It does support auto-imports, but only in API routes.